include/rosdiscover-clang/Ast/Stmt/SymbolicDeclRef.h

```diff
@@ -31,10 +32,18 @@ class SymbolicDeclRef : public SymbolicExpr {
   // Complex logic needed here to avoid duplication in sub-classes.
   ) : isInstanceMember(declRef->getDecl()->isCXXInstanceMember()), 
       isClassMember(declRef->getDecl()->isCXXClassMember()),
-      typeName(declRef->getType().getAsString()),
+      typeName(normalizeTypeName(declRef->getType().getAsString())),
       name(createName(declRef)),
       qualifiedName(declRef->getDecl()->getQualifiedNameAsString()) {}
 
+  static std::string normalizeTypeName(std::string clangTypeName) {
+    auto symbolicType = SymbolicValue::getSymbolicType(clangTypeName);
+    if (symbolicType == SymbolicValueType::Unsupported) {
+      return clangTypeName;
```

Comment:
It seems like we're losing information here since Clang types aren't real symbolic types and we lose the ability to distinguish them from actual symbolic types.